### PR TITLE
Using pybullet's humanoid.urdf

### PR DESCRIPTION
**Pull request by MarisaRipoll**
changed the urdf

### CODE REVIEW

File: bulletroboy/operator.py [hunk 1]
```diff
@@ -172,9 +172,9 @@ def two_end_effectors(self, positions, maxIter = 100, chest_constraint = True):
 		iter = 0
 
 		# Gets the hand link ids and the free revolute joints
-		_, freeJoints = self.get_EF_id(b'human/left_hand')
-		endEffectorIds = [self.op.get_link_index('human/left_hand'),
-                                  self.op.get_link_index('human/right_hand')]
+		_, freeJoints = self.get_EF_id(b'left_wrist')
```
Comment (missxa) [inline bulletroboy/operator.py:231]: why is the name of the link is hardcoded?

File: bulletroboy/operator.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def publish_state(self, ef_names = np.array(['human/left_hand','human/right_hand
 
 
 class Movements():
-	def __init__(self, operator, link = b'human/left_hand'):
+	def __init__(self, operator, link = b'left_wrist'):
```
Comment (missxa) [inline bulletroboy/operator.py:137]: the whole point of parameters is that you dont need to change the funtion implementation when some high level things are changed, like the urdf model...

**Comment by mlanas** [comment]
Movements class uses resetJointState which overrides pybullet's physics simulation, this will be change in another PR including the comments made.

File: bulletroboy/operator.py [hunk 3]
```diff
@@ -192,47 +248,56 @@ def two_end_effectors(self, positions, maxIter = 100, chest_constraint = True):
 	def simple_move(self, case):
```
Comment (karim-alweheshy) [inline bulletroboy/operator.py:248]: I think this method is too big, is there a way to split it?

Comment (mlanas) [reply bulletroboy/operator.py:248]: It's already smaller in the movement-class-fix branch, this PR is only about the human urdf file

**Review by NadaChatti — APPROVED** [review]

File: bulletroboy/operator.py (hunk 2 above)
Comment (NadaChatti) [reply bulletroboy/operator.py:137]: This would be handled in the upcoming utils/constants file where links (human + roboy) would be "standardized"

Comment (missxa) [reply bulletroboy/operator.py:137]: so i dont want to / not planning to review the same code multiple times. if you have a new version, where is it?

File: bulletroboy/operator.py (hunk 3 above)
Comment (missxa) [reply bulletroboy/operator.py:248]: so i dont want to / not planning to review the same code multiple times. if you have a new version, where is it?

**Pull request closed without merging**